isis/src/control/apps/jigsaw/jigsaw.xml

```diff
@@ -1513,5 +1513,106 @@
+         are two parameters which can only be solved for when ground points are present in the bundle solution since they are concerned with absolute positioning.
+         Applying the point radius solve parameter requires the point_radius_sigma to be set. This value is a representation the uncertainty (in meters) of the 
+         cameras pointing corresponding to the correct elevation on the shape model. Often this value can be set, and the appropriateness of the set value can be 
+         checked using the 'POINTS DETAIL' section of the bundleout.txt file output by jigsaw. If the radius total correction of points exceeds the provided sigma, 
+         the uncertainty may need to be increased.
+        </p>
+        <p>
+         Applying the space craft position solve parameter requires an uncertainty estimation through spacecraft_position_sigma, while [file_prefix]_bundleout_images.csv 
+         holds the resulting image adjustments in the X Correction, Y Correction, and Z Correction columns. The overhermite flag allows a polynomial to estimate the 
+         spacecraft position, like the overexisting flag estimates the camera pointing with a polynomial. As such, this option saves memory and time at the expense of 
+         some loss in accuracy and this loss can be mitigated by switching it off for the final run.
+        </p>
+      </description>
+      <terminalInterface>
+        <commandLine>
+         jigsaw fromlist=cubes_for_ground_update.lis cnet=grounded_relative_reg.net onet=grounded_rr_jig1.net update=no file_prefix=jig1rr sigma0=1.0e-10 maxits=10 \
+                twist=yes camsolve=accelerations overexisting=yes camera_angles_sigma=.25 camera_angular_velocity_sigma=.1 camera_angular_acceleration_sigma=.01 \
+                radius=yes point_radius_sigma=500 spsolve=positions overhermite=yes spacecraft_position_sigma=1000 
+        </commandLine>
+        <description>
+        </description>
+      </terminalInterface>
+    </example>
+
+    <example>
+      <brief>Run of jigsaw parameterized for Kaguya Terrain Camera (TC) images and <b>a final ground control</b> network covering Apollo 15 landing site. </brief>
+      <description>
+        <p>
+         This last example is of a final jigsaw run of a grounded network. In this example all network adjustments are done, the bundle is converging, the resulting 
+         residuals are acceptable, and therefore we are ready to update the camera pointing on the cubes. In the final run you can turn off overexisting and overhermite, 
+         if you want to regain the accuracy, but note that the bundle solution will differ from your previous run. Therefore, it is advisable to conduct one intermediate 
+         run to check the bundle is acceptable without these flags before updating the cubes. Finally, during the final run we turn on the error propagation flag. This 
```

Comment:
I would recommend not changing any parameters besides `update` in the final solution. You should check that you like the whole solution before you save it. It's kind of a hassle, but there's an issue open already to allow us to apply a previous solution independently. #4474